lib/active_model/serializer.rb

```diff
@@ -112,10 +112,25 @@ def self.digest_caller_file(caller_line)
       Digest::MD5.hexdigest(serializer_file_contents)
     end
 
+    # TODO(beauby): Cache this.
```

Comment:
Actually, caching here should not even be needed as the final serializer class will be cached.